src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -718,7 +770,14 @@ def validate_cfs_config(self, config_name: str):
                 self.assertIn('name', layer)
                 self.assertIn('commit', layer)
                 if self.cfs_version == 'v3':
-                    self.assertIn('clone_url', layer)
+                    try:
+                        self.assertIn('clone_url', layer)
+                    except AssertionError as assertion_err:
+                        try:
+                            self.assertIn('source', layer)
+                        except AssertionError as err:
+                            logging.error(f"Failed to find clone_url or source with errors: \n{assertion_err}\n{err}")
+                            raise
```

Comment:
I would suggest simplifying by just using `assertTrue`. Then you'd get the message you want in the actual assertion message as well, rather than having it as a separate error log message and just the second failed assertion about 'source' not being present in `layer`. Something like this:

```suggestion
                    self.assertTrue('clone_url' in layer or 'source' in layer,
                                    'Layer must contain either clone_url or source')
```